charts/temporal/templates/server-deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
       initContainers:
         {{- if $.Values.cassandra.enabled }}
         - name: check-cassandra-service
-          image: busybox
+          image:   "{{ $.Values.busyBox.image.repository }}:{{ $.Values.busyBox.image.tag }}"
```

Comment:
Please remove this change, it's unrelated to the image pull secrets.

Reply:
done